packages/react-dom/src/shared/CSSProperty.js

```diff
@@ -9,7 +9,9 @@
  * CSS properties which accept numbers but are not in units of "px".
  */
 export const isUnitlessNumber = {
+  animation: true,
```

Comment:
This one looks wrong to me. I assume it refers to the "duration" or "delay" parts of the shorthand? Looking at the [`animation-duration` docs](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-duration), it says:
> The time that an animation takes to complete one cycle. This may be specified in either seconds (s) or milliseconds (ms). The value must be positive or zero **and the unit is required**. A value of 0s, which is the default value, indicates that no animation should occur.

The [`animation-delay` docs](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-delay) say the same:
> The time offset, from the moment at which the animation is applied to the element, at which the animation should begin. This may be specified in either seconds (s) or milliseconds (ms). **The unit is required.**

Reply:
It's a bit hidden under the [grammar for `animation`](https://drafts.csswg.org/css-animations/#animation): `<time> || <easing-function> || <time> || <single-animation-iteration-count> || <single-animation-direction> || <single-animation-fill-mode> || <single-animation-play-state> || [ none | <keyframes-name> ]` where `<single-animation-iteration-count>` is valid syntax which can be a single number.

Reproducible in chrome: 
![animation-shorthand-unitless](https://user-images.githubusercontent.com/12292047/87176256-248b7600-c2da-11ea-88bf-1316a0e72560.png)

--https://codesandbox.io/s/animation-unitless-mg06t

Reply:
I see. I think changing `animation` still seems like a not-great idea since the majority of the numeric units this shorthand property covers do require units.

Reply:
Since `animation: 1px` is meaningless/invalid, I don’t think adding `animation` can break anything, at least (except if perhaps someone has a unitless number that gets px and thus ignored today).

Reply:
Yeah, I'm not familiar with this part of the code, so I don't know what the implications of adding this rule for a shorthand property that can contain multiple numeric values. I agree `animation: 1px` is meaningless though 😆